### ISSUE TITLE
Cannot set Client trust_env=False

### ISSUE THREAD

**Issue by addyess** [gtsystem/lightkube#19]
httpx client created [here](https://github.com/gtsystem/lightkube/blob/3d674f9f49ac31dd2f6ea0fbda5f996bfa2d16db/lightkube/config/client_adapter.py#L14-L28) doesn't allow override of the `trust_env` keyword option. 

Please consider some generic `**kwargs` for configuration of the client. `httpx.Client` will probably continue to expand its constructor [here](https://github.com/encode/httpx/blob/efc841a9e8e709f5cc40257927dd7461dc832330/httpx/_client.py#L612-L633)

**Comment by addyess** [comment]
Workaround ?

```python
    config = KubeConfig.from_file(kubeconfig_path)
    kubernetes = Client(config=config.get(), namespace=namespace)
    # Quick hack to see if `trust_env=False` on the httpx client will not use env proxy settings
    kubernetes._client._client.trust_env = False
    namespace_obj = Namespace(metadata=ObjectMeta(name=namespace))
    kubernetes.create(namespace_obj)

```

**Comment by gtsystem** [comment]
I don't have specific concerns to support the `trust_env` parameter (and you can create a PR for that), but I would like to avoid the generic `kwargs`: `lightkube` like `httpx` attempts to offer a clear and typed interface to end users, so that it's easier to use and more errors can be caught before runtime. Furthermore not all `httpx.Client` parameters make sense for a k8s client nor we can allow to override parameters that are generated from kubeconfig and needed for proper authentication.

**Comment by addyess** [comment]
i can agree with this style.  I will adjust my https://github.com/gtsystem/lightkube/pull/20 tomorrow to meet this request.  Thanks for the great library.

**Comment by addyess** [comment]
Updated.  thanks for the review @gtsystem

**Comment by gtsystem** [comment]
Released as part of v0.10.0
